website/events/templates/events/admin/registrations_table.html

```diff
@@ -62,7 +62,7 @@
                 <td data-sortval="{{ registration.date_cancelled|date:'c' }}">{{ registration.date_cancelled }}</td>
                 {% endif %}
                 {% for field in registration.information_fields %}
-                    {% if not field.value %}
+                    {% if not field.value and not field.field.type == 'boolean' %}
```

Comment:
I don't believe this is the change Thijs asked for. The thing is that the checkbox is also false when no value was entered. But we don't have a 'boolean' field type.

Reply:
> But we don't have a 'boolean' field type.
@se-bastiaan I first thought this was the case but the type of, what we assume is, 'checkbox' is actually of the type 'boolean' (try printing field.field.type). 

As for the change itself. I checked the root of this issue with a couple of other Technicie members from which I concluded that the way checkboxes are displayed in the CSV (e.g. False or null) should be mirrored in the admin panel. The problem (as described by Thijs) in my opinion asks for clarity in the admin panel whether a checkbox is False or null in stead of just showing an empty field.

If there is still any confusion feel free to PM me.

Reply:
I assumed based on [the line below](https://github.com/svthalia/concrexit/pull/1432/files#diff-0ffb59d377093550828358c44ac02d869263ae11f7c93e2b8d819c1baae6892bR67) that is was supposed to be checkbox :p 

Now that I have checked an actual event the value is indeed always true or false.

Maybe include that line in the changes?

Reply:
Yeah it's confusing, trust me I've bumped my head over that a couple of times :)

I'll update the changes!

Reply:
Yeah trust me, I've bumped my head over that issue a couple of times :)

I'll update the changes to the line to mentioned, changing the value from 'checkbox' to 'boolean' (if that's what you mean).

Reply:
That's exactly what I meant :)